test/CMakeLists.txt

```diff
@@ -343,6 +343,37 @@ if(MIOPEN_TEST_MLIR)
         COMMAND ${IMPLICITGEMM_MLIR_ENV_W} $<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_MLIR_ARGS_W} --input 128 64   56 56 --weights 64   64   1 1 --pads_strides_dilations 0 0 1 1 1 1 --in_layout NHWC --fil_layout NHWC --out_layout NHWC
         COMMAND ${IMPLICITGEMM_MLIR_ENV_W} $<TARGET_FILE:test_conv2d> ${IMPLICITGEMM_MLIR_ARGS_W} --input 256 1024 14 14 --weights 1024 32   1 1 --pads_strides_dilations 0 0 1 1 1 1 --group-count 32
     )
+
+    set(IMPLICITGEMM_MLIR_ENV_F_XDLOPS ${IMPLICITGEMM_MLIR_ENV_BASE} MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvMlirIgemmFwdXdlops)
+    set(IMPLICITGEMM_MLIR_ENV_B_XDLOPS ${IMPLICITGEMM_MLIR_ENV_BASE} MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvMlirIgemmBwdXdlops)
+    set(IMPLICITGEMM_MLIR_ENV_W_XDLOPS ${IMPLICITGEMM_MLIR_ENV_BASE} MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvMlirIgemmWrWXdlops)
+
+    # WORKAROUND: test_conv_igemm_mlir_xdlops target should run on gfx908 target only
+    # However, since it is a smoke target with MIOPEN_TEST_FLOAT, !MIOPEN_TEST_TENSILE, !SKIP_UNLESS_ALL flags
+    # It will be picked up by a non-gfx908 target. Therefore the need to add this if condition guard.
+    if(MIOPEN_TEST_GFX908)
+        add_custom_test(test_conv_igemm_mlir_xdlops
```

Comment:
The test is not ran for FP16. Please add `ALLOW_HALF`.

Reply:
Please also add `ALLOW_HALF` to `test_conv_igemm_mlir`.

Reply:
Wow, this is a pretty big hole. I'm really thankful that you managed to catch this at last minute. I should be more careful in checking the test running status in each test stage.

Reply:
FYI I have double checked that MLIR test with half/xdlops show up appropriately.

Reply:
Good thanks! This test/CmakeLists.txt is really badly designed (and I've participated in that).